### PR TITLE
[New] Add iris-ws w/ git auto-update via single package.json 

### PR DESCRIPTION
**Pull request by dibyendu**
Pull request for issue: #12884 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kataras/iris
 * Official website (optional, not the repository): https://iris-go.com/
 * License and its reference: https://github.com/kataras/iris/blob/master/websocket/LICENSE (BSD 3-Clause "New" or "Revised" License)
 * GitHub popularity (required):
   - Count of watchers: 547
   - Count of stars: 11109
   - Count of forks: 1141

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/iris-ws/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "filename": "client.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kataras/iris.git",
+    "fileMap": [
+      {
+        "basePath": "websocket",
+        "files": [
+          "client.js",
+          "client.min.js"
```
Comment (extend1994) [inline ajax/libs/iris-ws/package.json]: Let's use glob pattern `client*.+(js|map)` to replace line 12-13.

File: ajax/libs/iris-ws/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        ]
+      }
+    ]
+  },
+  "description": "Websocket client-side library for the golang web-framework called iris.",
```
Comment (extend1994) [inline ajax/libs/iris-ws/package.json]: May I know the reference of this description?

File: ajax/libs/iris-ws/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "author": {
+    "name": "Gerasimos Maropoulos",
+    "email": "kataras2006@hotmail.com"
+  },
+  "keywords": [
+    "iris",
+    "golang",
+    "websocket"
```
Comment (extend1994) [inline ajax/libs/iris-ws/package.json]: Maybe some more keywords from the upper plane of it git repo?

File: ajax/libs/iris-ws/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "license": "3-Clause BSD License",
```
Comment (extend1994) [inline ajax/libs/iris-ws/package.json]: Please use SDPX identifier `BSD-3-Clause`

File: ajax/libs/iris-ws/package.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "iris-ws",
+  "title": "iris-ws",
+  "filename": "client.js",
```
Comment (extend1994) [inline ajax/libs/iris-ws/package.json]: Let's use `client.min.js`

File: ajax/libs/iris-ws/package.json (hunk 2 above)
Comment (dibyendu) [reply ajax/libs/iris-ws/package.json]: I've wrote this line myself. If you feel like, you can take a look at the original web-socket library (which has been used as it is, in the **IRIS** web-framework) here: https://github.com/kataras/go-websocket

File: ajax/libs/iris-ws/package.json (hunk 1 above)
Comment (dibyendu) [reply ajax/libs/iris-ws/package.json]: done

File: ajax/libs/iris-ws/package.json (hunk 3 above)
Comment (dibyendu) [reply ajax/libs/iris-ws/package.json]: added a few more

File: ajax/libs/iris-ws/package.json (hunk 4 above)
Comment (dibyendu) [reply ajax/libs/iris-ws/package.json]: done

File: ajax/libs/iris-ws/package.json (hunk 5 above)
Comment (dibyendu) [reply ajax/libs/iris-ws/package.json]: done

**Pull request merged**